### DISCUSSION TITLE
The ability to hide the long

### DISCUSSION THREAD

**Discussion by diego-gz** [NickvisionApps/Parabolic#980] [Ideas]
It would be nice if there's a way to hide the log, so I can see more of the video list.

**Comment by nlogozzo** [comment]
Are you using Parabolic on Windows?

If so, this is coming in the next update I have redesigned the app's interface a little.

**Reply by diego-gz** [reply]
Thank you for the swift response, I will check out the next update 👍

**Reply by nlogozzo** [reply]
I will let you know once it's out!

**Comment by nlogozzo** [comment]
The redesign is here! https://github.com/NickvisionApps/Parabolic/releases/tag/2024.12.0
